### PR TITLE
feat: support binding to multiple traits

### PR DESCRIPTION
**Pull request by NOBLES5E**
Support binding a single provider to multiple traits.

### CODE REVIEW

File: dilib-macros/src/provide_attr.rs [hunk 1]
```diff
@@ -53,20 +53,22 @@ impl ProvideAttribute {
...
                     keys::BIND
                 )
-            });
-
-            // We need: Box<dyn TraitType + Send + Sync>
-            let boxed_type = format!("std::boxed::Box<dyn {} + Send + Sync>", type_string);
-
-            syn::parse_str::<Box<Type>>(&boxed_type)
-                .unwrap_or_else(|_| panic!("'{}' is not a valid trait type", type_string))
+            }).split(",") {
```
Comment (Neo-Ciber94) [inline dilib-macros/src/provide_attr.rs:64]: Use a local variable for the split value string before iterating

Comment (Neo-Ciber94) [inline dilib-macros/src/provide_attr.rs]: Use a local variable for the split value string before iterating

File: dilib-macros/src/provide_attr.rs [hunk 2]
```diff
@@ -147,8 +157,10 @@ impl ProvideAttribute {
             };
 
             // Let the rest of the code the same
-            #target
+            #result_code
+        }
         }
```
Comment (Neo-Ciber94) [inline dilib-macros/src/provide_attr.rs]: Wrong indentation, format the file

File: dilib-macros/src/provide_attr.rs (hunk 1 above)
Comment (NOBLES5E) [reply dilib-macros/src/provide_attr.rs]: done

File: dilib-macros/src/provide_attr.rs (hunk 2 above)
Comment (NOBLES5E) [reply dilib-macros/src/provide_attr.rs]: done

File: dilib-macros/src/provide_attr.rs (hunk 1 above)
Comment (NOBLES5E) [reply dilib-macros/src/provide_attr.rs]: @Neo-Ciber94 BTW I also sent an email to your hotmail 😄

**Comment by Neo-Ciber94** [comment]
@NOBLES5E pull the changes to re-run the pipeline

**Comment by NOBLES5E** [comment]
> @NOBLES5E pull the changes to re-run the pipeline

done

**Comment by Neo-Ciber94** [comment]
Merged, will fix later the error on the test suite

**Pull request merged**